README.md

```diff
@@ -58,17 +65,17 @@ If the chosen `db_password_strategy` is `aws-secrets-manager`, this terraform mo
 | k8s\_node\_affinity | Full node\_affinity object as per terraform/Kubernetes docs. | `object({})` | `{}` | no |
 | k8s\_node\_selector | Full node\_selector object as per terraform/Kubernetes docs. | `object({})` | `{}` | no |
 | k8s\_node\_tolerations | Full k8s\_node\_tolerations object as per terraform/Kubernetes docs. | `object({})` | `{}` | no |
-| k8s\_scheduler\_cpu | Total cpu to allot to the Beekeeper scheduler pod. | `string` | `"500m"` | no |
-| k8s\_scheduler\_ingress\_host | Ingress host name for Beekeeper path-scheduler. | `string` | `""` | no |
-| k8s\_scheduler\_ingress\_path | Ingress path regex for Beekeeper path-scheduler. | `string` | `""` | no |
-| k8s\_scheduler\_liveness\_delay | Liveness delay (in seconds) for the Beekeeper Scheduling service. | `number` | `60` | no |
-| k8s\_scheduler\_memory | Total memory to allot to the Beekeeper scheduler pod. | `string` | `"2Gi"` | no |
-| k8s\_scheduler\_port | Internal port that the Beekeeper Scheduler service runs on. | `number` | `8080` | no |
+| k8s\_scheduler\_apiary\_cpu | Total cpu to allot to the Beekeeper Scheduler Apiary pod. | `string` | `"500m"` | no |
+| k8s\_scheduler\_apiary\_ingress\_host | Ingress host name for Beekeeper Scheduler Apiary. | `string` | `""` | no |
+| k8s\_scheduler\_apiary\_ingress\_path | Ingress path regex for Beekeeper Scheduler Apiary. | `string` | `""` | no |
+| k8s\_scheduler\_apiary\_liveness\_delay | Liveness delay (in seconds) for the Beekeeper Scheduling service. | `number` | `60` | no |
+| k8s\_scheduler\_apiary\_memory | Total memory to allot to the Beekeeper Scheduler Apiary pod. | `string` | `"2Gi"` | no |
+| k8s\_scheduler\_apiary\_port | Internal port that the Beekeeper Scheduler Apiary service runs on. | `number` | `8080` | no |
 | message\_retention\_seconds | SQS message retention (s). | `string` | `"604800"` | no |
-| path\_scheduler\_docker\_image | Beekeeper path-scheduler image. | `string` | `"expediagroup/beekeeper-path-scheduler-apiary"` | no |
-| path\_scheduler\_docker\_image\_version | Beekeeper path-scheduler image version. | `string` | `"latest"` | no |
-| path\_scheduler\_ecs\_cpu | The amount of CPU used to allocate for the Beekeeper Path Scheduler Apiary ECS task.<br>Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | `string` | `"2048"` | no |
-| path\_scheduler\_ecs\_memory | The amount of memory (in MiB) used to allocate for the Beekeeper Path Scheduler Apiary container.<br>Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | `string` | `"4096"` | no |
+| scheduler\_apiary\_docker\_image | Beekeeper Scheduler Apiary image. | `string` | `"expediagroup/beekeeper-scheduler-apiary"` | no |
```

Comment:
README is missing `metadata_cleanup_docker_image_version` and `metadata_cleanup_docker_image`

Reply:
nice spot!